src/pages/publication/description_tab/ProjectSearch.tsx

```diff
@@ -1,38 +1,34 @@
 import React, { FC, useCallback, useEffect, useState } from 'react';
-import { useTranslation } from 'react-i18next';
 import { useDispatch } from 'react-redux';
 
-import { searchCristinProjects } from '../../../api/external/cristinProjectApi';
+import { searchProjects } from '../../../api/projectApi';
 import AutoSearch from '../../../components/AutoSearch';
-import { CristinProjectType, Project } from '../../../types/project.types';
+import { Project } from '../../../types/project.types';
 import useDebounce from '../../../utils/hooks/useDebounce';
 
 interface ProjectSearchProps {
   dataTestId: string;
   setValueFunction: (value: any) => void;
-  value: string;
   placeholder?: string;
 }
 
-const ProjectSearch: FC<ProjectSearchProps> = ({ dataTestId, setValueFunction, value, placeholder }) => {
+const ProjectSearch: FC<ProjectSearchProps> = ({ dataTestId, setValueFunction, placeholder }) => {
   const [searchResults, setSearchResults] = useState<Project[]>([]);
   const [searchTerm, setSearchTerm] = useState('');
   const [searching, setSearching] = useState(false);
 
   const debouncedSearchTerm = useDebounce(searchTerm);
   const dispatch = useDispatch();
-  const { t } = useTranslation();
 
   const search = useCallback(
     async (searchTerm: string) => {
       setSearching(true);
-      const response = await searchCristinProjects(`title=${searchTerm}`, dispatch);
+      const response = await searchProjects(`title=${searchTerm}`, dispatch);
       if (response) {
-        const normalizedResponse = response.map((project: CristinProjectType) => ({
-          id: project.cristin_project_id,
-          title: `${project.title[project.main_language]} (${project.main_language})`,
-        }));
-        setSearchResults(normalizedResponse);
+        const newMap = response.map((project: any) => {
+          return { ...project, title: project.titles[0].title };
```

Comment:
jeg får "Ingen treff" når jeg søker på "test" - men i nettverksloggen ser jeg at jeg får 5 treff. skjønner ikke at det du mapper ikke er riktig, for det ser riktig ut